navbar.css

```diff
@@ -0,0 +1,68 @@
+nav {
+    background: #ffffff;
+    opacity: 0.5;
+    width: 100%;
+    height: 75px;
+    position: fixed;
+    top: 0;
+    z-index: 1;
```

Comment:
dont think this is doing anything

Reply:
It should be fixing the navbar to the top with a white opaque bg. have tweaked and need z-index 2 to keep it visible whilst still having my rocket disappear behind the home image.